crates/ruff_python_formatter/src/expression/expr_attribute.rs

```diff
@@ -88,55 +83,54 @@ impl FormatNodeRule<ExprAttribute> for FormatExprAttribute {
                 value.format().fmt(f)?;
             }
 
-            if comments.has_trailing_own_line(value.as_ref()) {
-                hard_line_break().fmt(f)?;
-            }
```

Comment:
No longer necessary, since this is only possible if the node is parenthesized anyway.